pinot-core/src/main/java/org/apache/pinot/core/util/CrcUtils.java

```diff
@@ -80,6 +85,7 @@ public long computeCrc()
           checksum.update(buffer, 0, len);
         }
       }
+      LOGGER.info("Current file = {}, computed crc = {}", file, checksum.getValue());
```

Comment:
Can we move this out of the loop so that we don't flood the log for v1 segment? We can log the list of files and the final crc.
Maybe log this for MD5 as well?

Reply:
done